sdk/rust/examples/cli/src/main.rs

```diff
@@ -0,0 +1,18 @@
+#[tokio::main]
+async fn main() -> eyre::Result<()> {
+    let client = dagger_sdk::connect().await?;
+
+    let app_directory = client.host().directory("./app");
+
+    let build_file = client
+        .container()
+        .from("rust:slim-buster")
+        .with_directory("./app", app_directory)
+        .with_workdir("/app")
+        .with_exec(vec!["cargo", "build", "--release"])
+        .file("./target/release/app");
+
+    build_file.export("./build/cli").await?;
```

Comment:
not entirely sure export works if the folder doesn't already exists. I.e. `build`

Reply:
I tested the example on Ubuntu and MacOS.
The 'build' folder was on both OS created, if not present.